app/views/series/index.html.erb

```diff
@@ -20,8 +20,12 @@
       <% when_present(@recording_list) do |recording_list| %>
         <ul>
           <% recording_list.each do |recording| %>
-            <li class="duration-200 hover:text-red-800 text-base transform transition-colors" aria-label="<%= recording.title %>">
-              <%= link_to recording.title, recording.location %>
+            <li aria-label="<%= recording.title %>">
+              <% if recording.location.empty? %>
+                <p class="text-neutral-500"><%= recording.title %></p>
+              <% else %>
+                <%= link_to recording.title, recording.location, class: "duration-200 hover:text-red-800 text-base transform transition-colors underline" %>
+              <% end %>
```

Comment:
This is to avoid showing a "link" if there isn't a location available for the recording.